routes/mapper.py

```diff
@@ -164,21 +164,21 @@ def __init__(self, obj, resource_name=None, collection_name=None,
 
     def connect(self, *args, **kwargs):
         newkargs = {}
-        # newargs = args
-        routename, path = args
+        routename = args[0]
+        path = args[1] if len(args) > 1 else None
         for key, value in six.iteritems(self.kwargs):
             if key == 'path_prefix':
-                if len(args) > 1:
+                if path is not None:
```

Comment:
It wasn't a good idea to assign this value to `path` var as it's being modified in this loop